Description: Fix include paths
 Use more correct include paths, so -I. is not required.
Forwarded: yes, by private email; fixed upstream.
Last-Update: 2014-01-23

Index: cura-engine-14.01/utils/polygon.h
===================================================================
--- cura-engine-14.01.orig/utils/polygon.h	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/utils/polygon.h	2014-01-23 23:23:32.267784416 -0500
@@ -4,9 +4,9 @@
 #include <vector>
 #include <assert.h>
 using std::vector;
-#include "clipper/clipper.hpp"
+#include <clipper.hpp>
 
-#include "utils/intpoint.h"
+#include "intpoint.h"
 
 //#define CHECK_POLY_ACCESS
 #ifdef CHECK_POLY_ACCESS
Index: cura-engine-14.01/settings.h
===================================================================
--- cura-engine-14.01.orig/settings.h	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/settings.h	2014-01-23 23:25:47.000000000 -0500
@@ -1,7 +1,7 @@
 #ifndef SETTINGS_H
 #define SETTINGS_H
 
-#include <utils/floatpoint.h>
+#include "utils/floatpoint.h"
 #include <vector>
 
 #define VERSION "13.12"
Index: cura-engine-14.01/utils/floatpoint.h
===================================================================
--- cura-engine-14.01.orig/utils/floatpoint.h	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/utils/floatpoint.h	2014-01-23 23:23:32.267784416 -0500
@@ -7,7 +7,7 @@
 They represent millimeters in 3D space.
 */
 
-#include "utils/intpoint.h"
+#include "intpoint.h"
 
 #include <stdint.h>
 #include <math.h>
Index: cura-engine-14.01/modelFile/modelFile.h
===================================================================
--- cura-engine-14.01.orig/modelFile/modelFile.h	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/modelFile/modelFile.h	2014-01-23 23:23:32.267784416 -0500
@@ -9,8 +9,8 @@
 
 #include <vector>
 using std::vector;
-#include "utils/intpoint.h"
-#include "utils/floatpoint.h"
+#include "../utils/intpoint.h"
+#include "../utils/floatpoint.h"
 
 extern FILE* binaryMeshBlob;
 
Index: cura-engine-14.01/modelFile/modelFile.cpp
===================================================================
--- cura-engine-14.01.orig/modelFile/modelFile.cpp	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/modelFile/modelFile.cpp	2014-01-23 23:23:32.267784416 -0500
@@ -2,7 +2,7 @@
 #include <string.h>
 #include <stdio.h>
 
-#include "modelFile/modelFile.h"
+#include "modelFile.h"
 
 FILE* binaryMeshBlob = NULL;
 
Index: cura-engine-14.01/utils/logoutput.cpp
===================================================================
--- cura-engine-14.01.orig/utils/logoutput.cpp	2014-01-23 23:23:32.271784415 -0500
+++ cura-engine-14.01/utils/logoutput.cpp	2014-01-23 23:23:32.267784416 -0500
@@ -2,7 +2,7 @@
 #include <stdio.h>
 #include <stdarg.h>
 
-#include "utils/logoutput.h"
+#include "logoutput.h"
 
 int verbose_level;
 
Index: cura-engine-14.01/Makefile
===================================================================
--- cura-engine-14.01.orig/Makefile	2014-01-23 23:23:26.371785021 -0500
+++ cura-engine-14.01/Makefile	2014-01-23 23:23:43.403783277 -0500
@@ -6,7 +6,7 @@
 #g++ main.cpp modelFile/modelFile.cpp clipper/clipper.cpp -I. -o CuraEngine
 
 CXX ?= g++
-CFLAGS += -I. -c -Wall -Wextra -O3 -fomit-frame-pointer `pkg-config --cflags polyclipping`
+CFLAGS += -c -Wall -Wextra -O3 -fomit-frame-pointer `pkg-config --cflags polyclipping`
 # also include debug symbols
 #CFLAGS+=-ggdb
 LDFLAGS +=
